mitodl/bootcamp-ecommerce
# Add signatory name and signature customization fields in cms letter template page

#### What are the relevant tickets?
#818 

#### What's this PR do?
Add signatory name and signature customization options for cms letter template page

#### How should this be manually tested?
Just see cms template page has signatory field and check all the pages should show Mariah Rawding, signatory, for preview, acceptance/rejection letter and application letter

#### Screenshots
<img width="1431" alt="Screenshot 2020-08-13 at 11 30 35" src="https://user-images.githubusercontent.com/4043989/90101826-d514eb80-dd58-11ea-893f-14af075bf478.png">
<img width="1436" alt="Screenshot 2020-08-13 at 11 30 26" src="https://user-images.githubusercontent.com/4043989/90101819-d0503780-dd58-11ea-96c9-146d3e0453e2.png">
<img width="719" alt="Screenshot 2020-08-13 at 11 32 34" src="https://user-images.githubusercontent.com/4043989/90101830-d80fdc00-dd58-11ea-9c5e-f59b4b6744b8.png">
<img width="720" alt="Screenshot 2020-08-13 at 11 32 46" src="https://user-images.githubusercontent.com/4043989/90101833-d9410900-dd58-11ea-8db2-d64415d7c390.png">
<img width="1432" alt="Screenshot 2020-08-13 at 11 31 48" src="https://user-images.githubusercontent.com/4043989/90101828-d6461880-dd58-11ea-803b-b1e8282a1f36.png">
